### PR TITLE
Add user label to lts bucket

### PR DESCRIPTION
**Pull request by macox**
Signed-off-by: Mark Cox <markcox20@hotmail.com>

#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description

Add a `created-by` label to long term storage buckets for easier identification

### CODE REVIEW

File: pkg/cloud/gke/storage/long_term_storage.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ func createBucket(bucketName string, installValues map[string]string) (string, e
 	log.Logger().Infof("The bucket %s does not exist so lets create it", infoBucketURL)
 	region := gke.GetRegionFromZone(installValues[kube.Zone])
 	err := gke.CreateBucket(installValues[kube.ProjectID], bucketName, region)
+	gke.AddBucketLabel(bucketName, gke.UserLabel())
```
Comment (garethjevans) [inline pkg/cloud/gke/storage/long_term_storage.go]: On some of the linux distributions we have in cluster, the osUser.Current() returns nil, so we may need to handle the case where the label is blank

Comment (macox) [reply pkg/cloud/gke/storage/long_term_storage.go]: @garethjevans So an empty wouldn't be sufficient? (if found && label != "" {....})

**Pull request merged**